gpu/kinfu_large_scale/tools/CMakeLists.txt

```diff
@@ -31,17 +35,20 @@ source_group("Source Files" FILES ${srcs})
 
 PCL_ADD_EXECUTABLE(${the_target} COMPONENT ${SUBSYS_NAME} SOURCES ${srcs} ${hdrs} BUNDLE)
 target_link_libraries("${the_target}" pcl_common pcl_io ${OPENNI_LIBRARIES} pcl_visualization pcl_gpu_kinfu_large_scale pcl_octree)
+target_include_directories(${the_target} PRIVATE ${CUDA_TOOLKIT_INCLUDE})
```

Comment:
Might make sense to create an interface target (with that include source) and link that along with other libraries.

Reply:
Yeah. If that didn't work, let's move on :)